src/about/index.md

```diff
@@ -5,12 +5,12 @@ layout: layouts/page
 
 ## Founder History
 
-TODO: details about Modons' founders and origin
+Modons was born from the desire to make technology secure and accessible for everyone. During the pandemic in 2020, much of the world was forced online and the demand for computer assistance skyrocketed. After receiving many questions and concerns regarding the security of their digital lives, our customers suggested we provide organized services to help. Since then, we have had years of success and finally decided to make it official with the birth of Modons.com!
```

Comment:
```suggestion
Modons was born from the desire to make technology secure and accessible for everyone. During the pandemic in 2020, much of the world was forced online and the demand for computer assistance skyrocketed. After receiving many questions and concerns regarding the security of their digital lives, our customers suggested we provide organized services to help. Since then, we have had years of success and finally decided to make it official with the birth of Modons!
```